app/grandchallenge/core/utils/markdown.py

```diff
@@ -9,22 +12,64 @@ def extendMarkdown(self, md):  # noqa: N802
 
 
 class BS4Treeprocessor(Treeprocessor):
+
     def run(self, root):
+
+        el_class_dict = {
+            "img": "img-fluid",
+            "blockquote": "blockquote",
+            "table": "table table-hover table-borderless",
+            "thead": "thead-light",
+            "code": "codehilite",
+        }
```

Comment:
Now you're modifying all of these tags, it all needs tests which combine all of these blocks set either through markdown syntax or html directly.

Reply:
There is already a test for those tags/classes in markdown format. Do you recommend making a new test for the HTML case or modify the existing test? that test is not parametrized, so I need to modify parts of it. You told me before not to change other things, so I am checking beforehand.

Reply:
It is relevant to this changeset so change it.

Reply:
Done